doc/source/conf.py

```diff
@@ -1,14 +1,10 @@
 # -*- coding: utf-8 -*-
 from __future__ import division, absolute_import, print_function
+from functools import partial
 
 import sys, os, re
 
-# Check Sphinx version
-import sphinx
-if sphinx.__version__ < "1.2.1":
-    raise RuntimeError("Sphinx 1.2.1 or newer required")
-
-needs_sphinx = '1.0'
+needs_sphinx = '2.1'
```

Comment:
This isn't used in this file. Does Sphinx itself check this? If so, could you add a comment?

Reply:
Yup, it's a standard config variable (https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-needs_sphinx) and enforced:
```
Running Sphinx v1.8.5
1.18.dev0 1.18.0.dev0+066fd24

Traceback (most recent call last):
  File "/home/chris/anaconda3/lib/python3.7/site-packages/sphinx/cmd/build.py", line 303, in build_main
    args.tags, args.verbosity, args.jobs, args.keep_going)
  File "/home/chris/anaconda3/lib/python3.7/site-packages/sphinx/application.py", line 215, in __init__
    'be built with this version.') % self.config.needs_sphinx)
sphinx.errors.VersionRequirementError: This project needs at least Sphinx v2.1 and therefore cannot be built with this version.
```
Will add a comment